### PR TITLE
openssl: Add Light version and change 'openssl10' to full version

### LINKED ISSUE #147: OpenSSL 1.0.2 points to Light install
The openssl 1.0.2 package installs the "Light" version of the SSL lib, without compile headers etc. The openssl package from Main, however, installs the full version.

This difference is pretty confusing (and threw me off why builds stopped working when going back from 1.1.1 to 1.0.2).

Is there any reason 1.0.2 in Versions bucket shouldn't be the full distro rather than Light?

### PR DESCRIPTION
**Pull request by niheaven**
Fix #147

### CODE REVIEW

File: bucket/openssl-light.json [hunk 1]
```diff
@@ -15,9 +16,15 @@
             "hash": "69ae9e27c98a2ac87c210a22e1ab556ad65ab8b3a859b7f9cf1b109bd0530eac"
         }
     },
-    "innosetup": true,
+    "depends": "innounp",
+    "installer": {
+        "script": [
+            "Expand-InnoArchive -Path \"$dir\\$fname\"",
+            "Expand-InnoArchive -Path \"$dir\\$fname\" -DestinationPath \"$dir\\bin\" -ExtractDir '{cf}' -Removal",
```
Comment (Ash258) [inline bucket/openssl-light.json]: Why is this needed next to normal extraction?

Comment (niheaven) [reply bucket/openssl-light.json]: The default config file is in `{cf}` and will extract to `Common Files` during installation. So we should extract it and put them in the same folder as normal (not light) edition, i.e. `bin\cnf` (`{cf}\SSL`).

Normal edition also has `{cf}`, but it has `cnf` dir in `bin`, so there's no need to extract them. (See https://github.com/ScoopInstaller/Main/pull/251)
